starlette/websockets.py

```diff
@@ -181,6 +199,11 @@ async def close(
             {"type": "websocket.close", "code": code, "reason": reason or ""}
         )
 
+    async def send_response(self, response: Response) -> None:
```

Comment:
nit: I would call it `send_denial_response`, as it is a better match to the specs and makes it clear that it is a failure scenario.